src/backend/base/langflow/services/cache/factory.py

```diff
@@ -29,9 +29,9 @@ def create(self, settings_service: "SettingsService"):
                 logger.debug("Redis cache is connected")
                 return redis_cache
             logger.warning("Redis cache is not connected, falling back to in-memory cache")
-            return AsyncInMemoryCache()
+            return AsyncInMemoryCache(expiration_time=settings_service.settings.redis_cache_expire)
```

Comment:
```suggestion
            return AsyncInMemoryCache(expiration_time=settings_service.settings.cache_expire)
```

Reply:
This is Redis failure case so that inmemory cache should respect the Redis cache_expire.

Reply:
If you use `AsyncInMemoryCache` you're not using redis, therefore you should not use redis parameters.

The expire time is related to the implementation. Mixing the values can be dangerous.

In general I believe the current behaviour is wrong. If you configure to use redis but you messed up some connection config, langflow should stop instead of falling back. - but it's not related to this PR

Reply:
I agree. Let's merge this and we'll deactivate the fallback.

Reply:
Remove the fallback to AsyncCache behaviour since this can be confusing and make it difficult to debug if Redis cannot be connected at the initialization.